install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -103,3 +103,69 @@ spec:
             severity: warning
           annotations:
             message: "Memory usage of {{ $value | humanize }} on {{ $labels.node }} exceeds 90%. Master nodes starved of memory could result in degraded performance of the control plane."
+    - name: control-plane-cpu-utilization
+      rules:
+        - alert: HighOverallControlPlaneCPU
+          annotations:
+            summary: >-
+              CPU utilization across all three control plane nodes is higher than two control plane nodes can sustain; a single control plane node outage may
+              cause a cascading failure; increase available CPU.
+            runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/machine-config-operator/ExtremelyHighIndividualControlPlaneCPU.md
```

Comment:
Hmm, does anything substitute these references after we branch for example?  This URL doesn't exist right now, we'd need to move https://github.com/openshift/runbooks/blob/05b7f6fc49245bd12a6b223781d1a43f1e473541/alerts/cluster-kube-apiserver-operator/ExtremelyHighIndividualControlPlaneCPU.md - but if we did that I think the 4.10 URL would be broken: https://github.com/openshift/cluster-kube-apiserver-operator/blob/release-4.10/bindata/assets/alerts/cpu-utilization.yaml#L15

Reply:
We discussed having the runbook repository versioned at some point, but the idea was discarded. I don't recall exactly why, but I think it was because it would've been very painful to maintain.

The URL doesn't exist yet because I have a PR in flight to move it to the MCO directory: https://github.com/openshift/runbooks/pull/46. Since the URL change will occur in the 4.11/4.12 timeframe, it means that we will still have older versions with the previous URL so I added a symlink inside of the runbooks repository to prevent any dead links.